app/templates/macros/nav_macros.html

```diff
@@ -27,9 +27,9 @@
 {% endmacro %}
 
 {% macro account_items(current_user) %}
-    <a href="{{ url_for('main.about') }}" class="item">About</a>
```

Comment:
I think we can keep this About and Contact Us here and remove them from the if else block since they will be there regardless of if it is a user or admin